src/components/wallet/methods/updateAccount.ts

```diff
@@ -8,18 +8,34 @@ import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
 
 export default async function updateAccount(this: Wallet) {
+  function cbQuery({ this: wallet }) {
+    wallet.server
+      .claimableBalances()
+      .claimant(wallet.account.publicKey)
+      .call()
+      .then(function (resp) {
+        resp.records.forEach((record) => {
+          wallet.logger.response(
+            'Claimable Balances Available ',
+            loOmit(record, ['_links', 'paging_token', 'self'])
+          )
+        })
+        console.log(resp)
+      })
+      .catch(function (err) {
+        wallet.error = handleError(err)
+        console.error(err)
+      })
+    wallet.loading = { ...wallet.loading, update: false }
```

Comment:
I'm not a fan of setting `update: false` here when `update: true` is set in the outer function. I made a comment where `cbQuery` is called about how to change this.

Reply:
True your suggestion make much more sense